html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>html-forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html">
+    <label for="email">Email</label>
+    <div>
+      <input type="text" id="email" name="email" size="15" maxlength="30">
+    </div>
+    <div>
+      <label for="Why is Coding Cool?">Why is Coding Cool?</label>
+      <div>
+        <select name="reasons">
+          <option value="Select an option">Select an option</option>
+          <option value="Because it's interesting">Because it's interesting</option>
+          <option value="Because it's fun">Because it's fun</option>
+          <option value="Because it's powerful">Because it's powerful</option>
+        </select>
+      </div>
+    </div>
+    <div>
+      <p><label for="favoritelanguage">Favorite Language(choose one)</label></p>
+    </div>
+    <div>
+      <label><input type="radio" name="favoritelanguage" value="html" checked="checked" /> HTML </label>
+      <label><input type="radio" name="favoritelanguage" value="CSS" /> CSS </label>
+      <label><input type="radio" name="favoritelanguage" value="javascript" /> JavaScript </label>
+    </div>
+    <div>
+      <p><label for="moreinfo">Would you like to sign up for more information?</label></p>
+    </div>
+    <input type="checkbox" name="moreinfo" value="Sign me up!" checked="checked" /> Sign me up!
+    <div></div>
```

Comment:
make sure to use these empty divs to `wrap` the label and input elements, you don't want empty elements floating around.